### PR TITLE
update zh-trans /content/zh/docs/concepts/configuration/pod-priority-preemption.md

### PR DESCRIPTION
**Pull request by cCloudSky**
Add zh pod-priority-preemption.md

### CODE REVIEW

**Comment by cCloudSky** [comment]
I signed it

**Comment by tanjunchen** [comment]
@cCloudSky  Thanks for your contribution.
Please see https://github.com/k8smeetup/k8s-official-translation#k8smeetup-%E7%BF%BB%E8%AF%91%E6%B5%81%E7%A8%8B%E4%B8%8E%E7%BF%BB%E8%AF%91%E6%A0%A1%E7%A8%BF%E8%A7%84%E8%8C%83

see zh-translation example https://github.com/kubernetes/website/pull/19660/files for zh-trans format.
/retitle update zh-trans /content/zh/docs/concepts/configuration/pod-priority-preemption.md

File: content/zh/docs/concepts/configuration/pod-priority-preemption.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+如果一个Pod不能被调度，调度程序会尝试抢占（逐出）较低优先级的Pod，以便对挂起的Pod进行调度。
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+{{< warning >}}
+在集群中并非所有用户都是可以信任的，恶意用户可能会创建最高优先级的pod，从而导致其他Pod被驱逐/无法调度。
+管理员可以使用ResourceQuota阻止用户创建高优先级的Pod。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/configuration/pod-priority-preemption.md]: 中英文之间需要空格，具体翻译的格式要求参考上述链接

**Comment by cCloudSky** [comment]
必须加入k8smeetup组织吗 看了一下流程感觉非常繁琐

**Comment by markthink** [comment]
@cCloudSky  流程不复杂，加入是为了方便领取任务。

**Comment by tanjunchen** [comment]
@cCloudSky 
Thank you for your contribution. Some rules are set for better and continuous localization, and these rules are not very complicated.

**Pull request closed without merging**